src/components/modals/login-modal/LoginModal.tsx

```diff
@@ -0,0 +1,189 @@
+  );
+  const [email, setEmail] = useState<string>('');
+  const [password, setPassword] = useState<string>('');
+  const dispatch = useDispatch();
+
+  const setToForgotPasswordPage = () => {
+    setPage(ModalContent.ForgotPassword);
+  };
+
+  const getModalContent = () => {
+    switch (currentPage) {
+      case ModalContent.LoginContent:
+        return (
+          <ContentDiv>
+            <EmailInput
+              size={'large'}
+              placeholder="Email"
+              onChange={(e) => setEmail(e.target.value)}
+            />
+            <PasswordInput
+              size={'large'}
+              placeholder="Password"
+              onChange={(e) => setPassword(e.target.value)}
+            />
+            <ForgotPasswordText>
+              Forgot password?{' '}
+              <ForgotPasswordLinkButton onClick={setToForgotPasswordPage}>
+                Click here
+              </ForgotPasswordLinkButton>
+            </ForgotPasswordText>
+            <Text>
+              Don’t have an account?{' '}
+              <Link to="/signup" component={Typography.Link}>
+                Sign up here
+              </Link>
+            </Text>
+            {tokens.kind === AsyncRequestKinds.Failed && (
+              <Text>{tokens.error}</Text>
+            )}
+          </ContentDiv>
+        );
+      case ModalContent.ForgotPassword:
+        return (
+          <ContentDiv>
+            <Text>Enter email address associated with account</Text>
+            <EmailInput size="large" placeholder="Email" />
+          </ContentDiv>
+        );
+      case ModalContent.ResetPassword:
+        return (
+          <ContentDiv>
+            <Text>
+              Keep an eye on your inbox (and check your spam folder as well). If
+              you still haven’t received an email,{' '}
+              <Typography.Link>click here</Typography.Link> to resend it.
+            </Text>
+          </ContentDiv>
+        );
+    }
+  };
+
+  const getTitle = (): string => {
+    switch (currentPage) {
+      case ModalContent.LoginContent:
+        return 'Log in to your account';
+      case ModalContent.ForgotPassword:
+        return 'Reset your password';
+      case ModalContent.ResetPassword:
+        return 'Your password reset link is on the way!';
+    }
+  };
+
+  const handleOk = (): void => {
+    switch (currentPage) {
+      case ModalContent.LoginContent:
+        dispatch(login({ email, password }));
+        break;
+      case ModalContent.ForgotPassword:
+        setPage(ModalContent.ResetPassword);
+        break;
+      case ModalContent.ResetPassword:
+        onCloseLoginModal();
+        break;
+    }
+  };
+
+  const getOkText = (): string => {
+    switch (currentPage) {
+      case ModalContent.LoginContent:
+        return 'Sign In';
+      case ModalContent.ForgotPassword:
+        return 'Send Link';
+      case ModalContent.ResetPassword:
+        return '';
+    }
+  };
+
+  return (
+    <div className="modal">
```

Comment:
Two things: 
1. remove className
2. remove the `login-modal.less` file, it's empty anyways